src/client/Microsoft.Identity.Client/AppConfig/ConfidentialClientApplicationBuilder.cs

```diff
@@ -269,22 +269,19 @@ public ConfidentialClientApplicationBuilder WithClientAssertion(Func<AssertionRe
         }
 
         /// <summary>
-        /// Instructs MSAL.NET to use an Azure regional token service.
+        /// Instructs MSAL.NET to use an Azure regional token service. This feature is currently available to 
+        /// first party applications only. 
         /// </summary>
         /// <param name="azureRegion">Either the string with the region (preferred) or        
         /// use <see cref="ConfidentialClientApplication.AttemptRegionDiscovery"/> and MSAL.NET will attempt to auto-detect the region.                
         /// </param>
         /// <remarks>
-        /// Region names as per https://azure.microsoft.com/en-ca/global-infrastructure/geographies/.
-        /// See https://aka.ms/region-map for more details on region names.

```

Comment:
Why remove this?

Reply:
It doesn't show the actual short region names. See customer issue associated with PR.